<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>image</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="grbox_action">
        <property name="title">
         <string>Actions</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btn_seuillage">
           <property name="text">
            <string>Seuillage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_add">
           <property name="text">
            <string>Addition</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_soust">
           <property name="text">
            <string>Soustraction</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_erosion">
           <property name="text">
            <string>Erosion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_dilatation">
           <property name="text">
            <string>Dilatation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ouverture">
           <property name="text">
            <string>Ouverture</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_fermeture">
           <property name="text">
            <string>Fermeture</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_amin">
           <property name="text">
            <string>Amincissement</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_epai">
           <property name="text">
            <string>Epaississement</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grbox_option">
           <property name="title">
            <string>Option</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="btn_squelette">
                <property name="text">
                 <string>Squelette</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>It√©rations</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox"/>
              </item>
              <item>
               <widget class="QPushButton" name="btn_amin_homtho">
                <property name="text">
                 <string>Amincissement homothopique</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="btn_reset">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_quitte">
                <property name="text">
                 <string>Quitter</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuouvrir">
    <property name="title">
     <string>Ouvrir</string>
    </property>
    <addaction name="actionfichier"/>
   </widget>
   <widget class="QMenu" name="menuenregister">
    <property name="title">
     <string>Enregistrer</string>
    </property>
    <addaction name="actionenregistrer_sous"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionNiveau_de_gris"/>
   </widget>
   <addaction name="menuouvrir"/>
   <addaction name="menuenregister"/>
   <addaction name="menuImage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionfichier">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fichier</string>
   </property>
  </action>
  <action name="actionenregistrer_sous">
   <property name="text">
    <string>Enregistrer sous</string>
   </property>
  </action>
  <action name="actionNiveau_de_gris">
   <property name="text">
    <string>Niveau de gris</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
